lib/templates.js

```diff
@@ -173,31 +177,40 @@ const createAPIResourceTemplate = (serviceName, serviceDefinition, appConfig, pa
 
             ...ordExtensions,
         };
-
-        apiResources.push(obj);
+        // visibility can also be set as an ORD extension; in this case, it overwrites the default visibility
+        if (obj.visibility === RESOURCE_VISIBILITY.public) apiResources.push(obj);
```

Comment:
I would suggest we move this logic to other place. e.g. ord.js. And create a function might be so called: `filterNotPublicObj()`. The reasons:
- apiResources and eventResources can use the same function
- I'm not sure in the future, should we consider we hide the information in production env, but output it in dev or test env?

Reply:
@aramovic79 , I assume you mean something different than @zongqichen . Having a consistent filter function like "isVisible(ordResource)" might be useful, esp. if we introduce additional logic around the filtering, e.g. if correctly authenticated, then also show internal and private resources... such additions can than be implemented in such a central function. 
Anyhow I wouldn't overthink it now, you can do such refactoring also later.